app/controllers/user_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class UserController < ApplicationController
+  def new; end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome #{@user} !"
+      sign_in @user
+      redirect_to @user
+    else
+      flash.now[:danger] = 'invalid inputs'
+      render 'new'
+    end
+  end
+
+  def show
+    @user = User.find_by(id: params[:id])
```

Comment:
why not use `@current_user` ?
![Screenshot from 2019-12-11 01-13-04](https://user-images.githubusercontent.com/8848104/70580258-67a23100-1bb3-11ea-826b-1a7c25a46883.png)

Reply:
Fixed, @current_user is now used